clash-prelude/src/Clash/Prelude/BlockRam/File.hs

```diff
@@ -31,6 +32,12 @@ For example, a data file @memory.bin@ containing the 9-bit unsigned number
 000001101
 @
 
+Such a file can be produced with 'memFile':
```

Comment:
```suggestion
Such a file can be produced with 'E.memFile':
```
should fix the haddock error

Reply:
Check. Were that all the errors in CI? I can't see the old result anymore since I force pushed.

Reply:
All the haddock errors at least.

Reply:
Okay, tnx. Then let's see how the latest one fares.